docs/more-info/build-server-setup/tfs-build-vnext.md

```diff
@@ -1,16 +1,29 @@
 # Visual Studio Online (Build vNext) Setup
 ## Basic Usage
-In [Visual Studio Online](https://www.visualstudio.com/) build vNext (the web based build system) you can add GitVersion as a Build Step. This requires a one-time setup to import the GitVersion task into your VSO instance.
+In [Visual Studio Online](https://www.visualstudio.com/) build vNext (the web based build system) you can call GitVersion either using the Command Line build step or install a custom build step. This requires a one-time setup to import the GitVersion task into your VSO instance.
 
-## Installing/updating the VSO Build Step
+## Using GiVersion with the MSBuild Task NuGet Package
+1. Add the [GitVersionTask](https://www.nuget.org/packages/GitVersionTask/) NuGet Package to your projects.
+
+See [MSBuild Task](http://gitversion.readthedocs.org/en/latest/usage/#msbuild-task) for further instructions how to use the MS Build Task.
+
+## Using the GitVersion with the Command Line build step
+1. Make sure to have GitVersion.exe under version control.
```

Comment:
Would this be the correct chocolatey package: https://chocolatey.org/packages/GitVersion.Portable?